docs/modules/ROOT/pages/client-auth.adoc

```diff
@@ -0,0 +1,109 @@
+= Client Authentication and Authorization
+
+Use the Hazelcast Platform Operator to configure xref:hazelcast:security:authentication-overview.adoc[Client Authentication] and xref:hazelcast:security:client-authentication.adoc[Client Authentication].
```

Comment:
```suggestion
Use the Hazelcast Platform Operator to configure Client Authentication. For more information, see: xref:hazelcast:security:authentication-overview.adoc[Client Authentication overview] and xref:hazelcast:security:client-authentication.adoc[Client Authentication].
```

Reply:
I tweaked the above otherwise it would state 'Client Authentication and Client Authentication'

Reply:
Sorry, it was my fault. I fixed it, it should make sense now.